src/shared/util.js

```diff
@@ -798,7 +824,11 @@ class Util {
   // This calculation uses Singular Value Decomposition.
   // The SVD can be represented with formula A = USV. We are interested in the
   // matrix S here because it represents the scale values.
-  static singularValueDecompose2dScale([m0, m1, m2, m3], output) {
+  static singularValueDecompose2dScale(matrix, output) {
+    const m0 = matrix[0];
+    const m1 = matrix[1];
+    const m2 = matrix[2];
+    const m3 = matrix[3];
```

Comment:
Should we also change the following spots in the same way, to improve consistency?
 - https://github.com/mozilla/pdf.js/blob/16c876569f134081431602d6c1dd073381500e86/src/shared/util.js#L745
 - https://github.com/mozilla/pdf.js/blob/16c876569f134081431602d6c1dd073381500e86/src/shared/util.js#L751
 - https://github.com/mozilla/pdf.js/blob/16c876569f134081431602d6c1dd073381500e86/src/shared/util.js#L761